packages/amplify_api_plugin_interface/lib/src/graphql/graphql_request.dart

```diff
@@ -13,19 +13,27 @@
  * permissions and limitations under the License.
  */
 
-import '../uuid.dart';
+// TODO: Datastore dependencies temporarily added in API. Eventually they should be moved to core or otherwise reconciled to avoid duplication.
+import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
+
+// TODO: Remove alias when Datastore dependency is removed
+import '../uuid.dart' as API_UUID;
 
 class GraphQLRequest<T> {
   final String? apiName;
-  final String document;
-  final Map<String, dynamic> variables;
-  final String cancelToken = UUID.getUUID();
 
-  GraphQLRequest({
-    this.apiName,
-    required this.document,
-    this.variables = const <String, dynamic>{},
-  });
+  String document;
```

Comment:
should add final keyword here I suppose, my bad, messed this up during conflict resolution